_pkgdown.yml

```
@@ -1,61 +1,48 @@
+url: epiforecasts.io/scoringutils/
```

```suggestion
url: https://epiforecasts.io/scoringutils/
```

This needs to be a valid URL otherwise the sitemap won't be correct.

---

thanks!